src/gui/MainWindow.cc

```diff
@@ -1314,6 +1314,7 @@ void MainWindow::compileEnded()
   clearCurrentOutput();
   GuiLocker::unlock();
   if (designActionAutoReload->isChecked()) autoReloadTimer->start();
+  emit compilationDone(this->rootFile);
```

Comment:
It is a bit confusing to have both compileDone and compilationDone signals. Ideally, we should refactor this to make the flow less compex, but that's a bit out of scope. In the meantime, perhaps tag this signal (#ifdef or code comment) as a GUI-test feature so other people don't start abusing it.